lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -800,36 +800,36 @@ def get_localhosts(self):
     def make_mpdboot_options(self):
         """Add various options to mpdboot_options"""
 
-        self.mpdboot_options = self.MPDBOOT_OPTIONS[:]
+        self.mpdboot_options = CmdList(self.MPDBOOT_OPTIONS)
```

Comment:
I think that copy of self.MPDBOOT_OPTIONS was there for a reason, better keep this a copy?

Reply:
The copy is there to avoid that `self.MPDBOOT_OPTIONS` gets changed in-place when we add stuff to `self.mpbdoot_options`, but that's no longer required when using `CmdList` (the copying is done implicitly there).